wagtailmenus/__init__.py

```diff
@@ -2,7 +2,7 @@
 
 # major.minor.patch.release.number
 # release must be one of alpha, beta, rc, or final
-VERSION = (3, 1, 9, "final", 0)
+VERSION = (3, 2, 0, "rc", 0)
```

Comment:
```suggestion
VERSION = (4, 0, 0, "rc", 0)
```
I think we can call this v4.0... By the way, don't forget to update the `CHANGELOG.md` and the documentation's release notes in this PR  😉

https://wagtailmenus.readthedocs.io/en/stable/releases/index.html

Reply:
May I suggest the following CHANGELOG entry for my recent contribution:

```
* Fix [#459](https://github.com/jazzband/wagtailmenus/issues/459) by adding support for Wagtail 5.2, Wagtail 6.0 and Python 3.12.
```

Reply:
I would just go with "Added support for x, y, z" and then add another entry saying 

```md
Fix [#459](https://github.com/jazzband/wagtailmenus/issues/459) by migrating to wagtail.snippets. Removed `wagtail_modeladmin` support.
```

That way it becomes more explicit.